geo/benches/sweep_line_intersection.rs

```diff
@@ -177,7 +148,6 @@ fn generate_random_line(rng: &mut impl Rng) -> Line<f64> {
 // Benchmark edge cases
 fn bench_essential_edge_cases(c: &mut Criterion) {
     let mut group = c.benchmark_group("Essential Edge Cases");
-    group.sample_size(10);
```

Comment:
Using the default (larger) sample size made these benches a little more stable. These particular tests were fast enough to get a good sampling in the 5s warmup. 

There indeed were some slower ones - I've left the custom sample size for those.